fdbserver/QuietDatabase.actor.cpp

```diff
@@ -124,8 +124,18 @@ int64_t getQueueSize( const TraceEventFields& md ) {
 	return inputBytes - durableBytes;
 }
 
+//Computes the popped version lag for tlogs
+int64_t getPoppedVersionLag( const TraceEventFields& md ) {
+	int64_t persistentDataDurableVersion, queuePoppedVersion;
+
+	sscanf(md.getValue("PersistentDataDurableVersion").c_str(), "%lld", &persistentDataDurableVersion);
```

Comment:
```suggestion
	persistentDataDurableVersion = boost::lexical_cast<long long>(md.getValue("PersistentDataDurableVersion"));
```

is the more C++ way to do this.  You'll need to `#include <boost/lexical_cast.hpp>` at the top.